client/src/pages/chart.test.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import {mount, shallow} from 'enzyme'
+import ChartComponent from './chart'
+import {act} from 'react-dom/test-utils'
+
+
+jest.mock('../services/watchlistService', () => {
+    return {
+        __esModule: true,
+        getWatchlist: async() => [
+            {
+                ticker: "MICR",
+                company_name: "MICRON SOLUTIONS INC",
+                last_price: "50.40",
+                price_per_share: "50.40",
+                date_added: "2021-03-02T01:22:47.086Z"
+            }
+        ]
+    }
+})
+
+jest.mock('../services/portfolioService', () => {
+    return {
+        __esModule: true,
+        getPortfolioItems: async() => [
+            {
+                symbol: "MSFT",
+                company_name: "MICROSOFT CORP",
+                quantity: "50",
+                price_per_share: "100",
+                date_changed: "2021-03-02T01:22:47.086Z"
+            }
+        ]
+    }
+})
+
+
+describe('ChartComponent tests', () => {
+    require('../services/portfolioService');
```

Comment:
If this whole suite is blank, can we comment it out?

Reply:
hey so if commenting that out would make the test fail :( 
We could instead delete the chart.test.js file, but I wanted to keep it for later use, which is why I left some minimal code that wouldn't cause the test to fail :D

Reply:
Ahh gotcha, that's okay then!